setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = s2geometry
+version = 0.10.0
+description = Python packaging of s2geometry
+author = Brian Miles
+author_email = selimnairb@gmail.com
+license= MIT
```

Comment:
license is Apache-2.0, but it looks like using `classifiers` is recommended:
Update to latest google3 version